Util/team_color_service.py

```diff
@@ -4,18 +4,18 @@
 from config.config_service import ConfigService
 
 
-class TeamColorService(object, metaclass=Singleton):
-    BLUE = "blue"
-    YELLOW = "yellow"
+class TeamColorService(metaclass=Singleton):
+    BLUE = 'blue'
+    YELLOW = 'yellow'
 
     def __init__(self):
-        cfg = ConfigService()
+        cfg = ConfigService().config_dict
```

Comment:
We could instead of importing config_dict, directly access the config dict by overloading the __getitems__.

Reply:
Bonne idée!